source/release-notes/backend.txt

```diff
@@ -13,6 +13,30 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240605:
+
+5 June 2024 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Support for nested collections in ``Mixed`` properties is now available. To learn more, refer to :ref:`<collections-in-mixed>`. 
+
+- Improvements to the Function Editor for all Apps, including the following changes:
+
+  - A new :guilabel:`Format Code` button to format function code.
+
+  - A new :guilabel:`Clear & Run` button that replace the :guilabel:`Clear Result` button.
+  
+  - Autocomplete for common functionalities and App Services context.
+
+- App error logs now include GraphQL failures.
+
+- Users can no longer disable client max offline time. Users can continue to increase or decrease
```

Comment:
just confirming again whether we have a corresponding DOCSP ticket for the doc updates on our end

Reply:
https://jira.mongodb.org/browse/DOCSP-40390

Reply:
thanks!!